### PR TITLE
Use json ppm

### PR DESCRIPTION
**Pull request by coolo**

### CODE REVIEW

File: backend/baseclass.pm [hunk 1]
```diff
@@ -406,16 +407,15 @@ sub enqueue_screenshot {
...
 
     if ($sim > 50) {    # we ignore smaller differences
         $self->write_encoder_frame('R');
         $watch->lap("write_encoder_frame R");
     }
     else {
-        $self->write_img($image, $filename) || die "write $filename";
-        $self->write_encoder_frame("E $filename");
+        #$self->write_img($image, $filename) || die "write $filename";
+        #$self->write_encoder_frame("E $filename");
```
Comment (okurz) [inline backend/baseclass.pm]: Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit

File: t/11-image-ppm.t [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+BEGIN {
+    unshift @INC, '..';
+}
+
+use OpenQA::Benchmark::Stopwatch;
+
+
+# optional but very useful
+eval 'use Test::More::Color';                 ## no critic
+eval 'use Test::More::Color "foreground"';    ## no critic
```
Comment (okurz) [inline t/11-image-ppm.t]: sounds nice but I can't see a difference when trying it out locally on another test.

File: backend/baseclass.pm (hunk 1 above)
Comment (coolo) [inline backend/baseclass.pm]: I guess I should have said it in the description: but this *is* my local git repository. I just happened to create a PR from it.

File: t/11-image-ppm.t (hunk 2 above)
Comment (coolo) [inline t/11-image-ppm.t]: you mean colours? It's no reason to be ashamed. Happens to many

Comment (coolo) [inline t/11-image-ppm.t]: it also renders passed as green

**Pull request merged**